src/NewsService.Models.Dto/Responses/NewsResponse.cs

```diff
@@ -10,7 +10,6 @@ public class NewsResponse
         public string Subject { get; set; }
         public User Author { get; set; }
         public User Sender { get; set; }
-        public DateTime CreatedAt { get; set; }
```

Comment:
i guess we need it here, and creater too

Reply:
I think we should discuss this with the frontend